django_project/ford3/views/qualification_wizard.py

```diff
@@ -85,12 +87,35 @@ def validate_form_data(self, form_data):
             **qualification_fields
         )
 
+        # Subjects
+        post_dict = self.request.POST
+        subject_length = post_dict.get('subject-length', 1)
+        for subject_index in range(int(subject_length)):
+            subject = None
+            subject_index += 1
```

Comment:
I don't understand why you manipulate the index?
Does it help if the range starts from 1?